src/aioswitcher/api/__init__.py

```diff
@@ -562,6 +563,34 @@ async def stop(self) -> SwitcherBaseResponse:
         return SwitcherBaseResponse(response)
 
 
+@final
```

Comment:
Wouldn't it be better to have just one API class that takes the TCP port in its constructor?
Even and enum would be good I believe as a port identifier.

And if not, can we think of a title to better categorize _plugs & boilers_ vs. _thrmostats & runners_ instead of _type1_ vs. _type2_?

This just sounds too weird, if it's mandatory maybe consider boiler-plating this by creating similar classes to identify various types?